src/mcp_server_helm/commands/package.py

```
@@ -0,0 +1,146 @@
+import logging
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)